translations/README_vi_VN.md

```diff
@@ -15,6 +15,19 @@
 </p>
 
 
+# Translations
```

Comment:
"Translations" => "Đọc bằng ngôn ngữ khác"

Reply:
I thought about leaving this in english for the reason that if you accidentally come to the language you should still read this I would be willing to put it in the national languages in brackets behind it but in general I had it just like the disclaimer thought to just leave this in english